sherpa/astro/io/pyfits_backend.py

```diff
@@ -78,23 +79,27 @@ def _has_hdu(hdulist, name):
     return True
 
 
-def _try_key(hdu, name, fix_type=False, dtype=SherpaFloat):
+def _try_key(hdu, name, *, fix_type=False, dtype=SherpaFloat):
 
     try:
         key = hdu.header[name]
     except KeyError:
         return None
```

Comment:
```suggestion
    key = hdu.header.get(name, None)
```

Reply:
doh,

Reply:
Actually it needs to be

```suggestion
    key = hdu.header.get(name, None)
    if key is None:
        return None
```

and it also suffers from the poorly-named variable, as `name` is the "key"; this is more like value but that's a larger change to make.